.travis.yml

```diff
@@ -2,6 +2,11 @@ dist: trusty
 language: java
 before_install:
   - rm ~/.m2/settings.xml
+  - git clone https://github.com/cmebarrow/nukleus-kafka.spec
+  - cd nukleus-kafka.spec
+  - git checkout fix_build_failure
+  - mvn clean install -DskipTests
+  - cd ..
```

Comment:
Are there any spec changes required?

Reply:
Just for a sporadic test failure that caused the Travis build for the PR to fail.   Unrelated to the changes to increase message cache size,  but I just wanted to make sure the build  is as stable as possible.